### DISCUSSION TITLE
Fermata can work on Android Auto but no Youtube option

### DISCUSSION THREAD

**Discussion by LasGeo64** [AndreyPavlenko/Fermata#110] [Q&A]
My phone is pixel 6 pro with android12 .
AA version is 7.0.614134.
I installed fermata v1.8.1 from android play store.
I can open fermata in AA but no YT option.
YT addon has been enabled.
Can anyone comment how to make it work?
Thanks.
![Screenshot_20211217-175827](https://user-images.githubusercontent.com/96324865/146627520-09234322-5e38-46df-bf1d-4e45570e0aab.png)
![PXL_20211217_103252126](https://user-images.githubusercontent.com/96324865/146627521-f6a66962-8afe-42cb-b780-8e7ea6fd710e.jpg)

**Comment by AndreyPavlenko** [comment]
What you open in AA is not Fermata, but a built-in AA media player and it does not support video.
If you want youtube in AA, you need to install unofficial version - Fermata Auto - https://forum.xda-developers.com/t/fermata-media-player-audio-and-video-player-for-android-auto.4079519/.
But, for Android 12, you need to downgrade AA to 6.9.

**Reply by LasGeo64** [reply]
Thanks  Andrey for the detailed explanation.
I will have a trial.
And really appreciate you for free sharing.

**Reply by rml-augustine** [reply]
> Thanks Andrey for the detailed explanation. I will have a trial. And really appreciate you for free sharing.

is it worked?
